pkg/shared/families/infofinder/types/scannerResult.go

```diff
@@ -0,0 +1,41 @@
+// Copyright © 2022 Cisco Systems, Inc. and its affiliates.
```

Comment:
I think It is better to follow the convention in the other families. This file should be under common/results.go

Reply:
I thought I followed the convention since I did it the same as in the `misconfiguration` familiy. It looks like we don't have a convention :)

Reply:
We probably need to have a clean up across all the familes. From my experience having a "types" package which holds all the type declarations and their helper functions is a pretty standard thing in golang.

Reply:
I agree